README.md

```diff
@@ -3,15 +3,17 @@
 A curated list of examples related to actix.
 
 ## from community
-* [kriry.com](http://kriry.com/) : Explore-Interactive net. [source](https://github.com/kriry/waler)
-* [Roseline](https://github.com/DoumanAsh/roseline.rs) : A personal web site and discord & IRC bot to access simple SQLite database. Demonstrates usage of various actix and actix-web concepts.
-* [Actix Auth Server](https://hgill.io/posts/auth-microservice-rust-actix-web-diesel-complete-tutorial-part-1/) : Auth web micro-service with rust using actix-web - complete tutorial. See code in [examples/simple-auth-server](https://github.com/actix/examples/tree/master/simple-auth-server)
-* [actix-raft](https://github.com/railgun-rs/actix-raft) : An implementation of the Raft consensus protocol using the actix Actor framework.
-* [lemmy](https://github.com/dessalines/lemmy) : A federated alternative to reddit in rust.
-* [actix-realworld-example-app](https://github.com/fairingrey/actix-realworld-example-app) : Implementation of the RealWorld backend API spec in Actix.
-* [Canduma](https://github.com/clifinger/canduma) : rust authentication server boilerplate
-* [Broker](https://crates.io/crates/broker) : Real-time Zero-Code API Server
-* [Rust, Docker & GraphQL](https://github.com/jayy-lmao/rust-graphql-docker): An example of using Dataloaders, context, and a minimal docker container. 
+
+- [kriry.com](http://kriry.com/) : Explore-Interactive net. [source](https://github.com/kriry/waler)
+- [Roseline](https://github.com/DoumanAsh/roseline.rs) : A personal web site and discord & IRC bot to access simple SQLite database. Demonstrates usage of various actix and actix-web concepts.
+- [Actix Auth Server](https://hgill.io/posts/auth-microservice-rust-actix-web-diesel-complete-tutorial-part-1/) : Auth web micro-service with rust using actix-web - complete tutorial. See code in [examples/simple-auth-server](https://github.com/actix/examples/tree/master/simple-auth-server)
+- [actix-raft](https://github.com/railgun-rs/actix-raft) : An implementation of the Raft consensus protocol using the actix Actor framework.
+- [lemmy](https://github.com/dessalines/lemmy) : A federated alternative to reddit in rust.
+- [actix-realworld-example-app](https://github.com/fairingrey/actix-realworld-example-app) : Implementation of the RealWorld backend API spec in Actix.
+- [Canduma](https://github.com/clifinger/canduma) : rust authentication server boilerplate
+- [Broker](https://crates.io/crates/broker) : Real-time Zero-Code API Server
+- [Rust, Docker & GraphQL](https://github.com/jayy-lmao/rust-graphql-docker): An example of using Dataloaders, context, and a minimal docker container.
```

Comment:
I think we shouldn't change the rest of the markdown file so we can keep the git history cleanner.

Reply:
Yea, sorry, auto-format on save. I'll turn it off and revert.

Reply:
Reverted